README.md

```diff
@@ -29,3 +30,5 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+Answer: In terms of number of comparisons, the time complexity would be $\Theta(n^2)$. In terms of the number of flips, the time complexity would be $\Theta(n^2)$.
```

Comment:
I think I remember what I was thinking. The flips themselves would be 2n but the flip function itself iterates through the array as n decreases which would be n/2, so then it would be (n/2) * n + (n/2)* n which is 2n^2/2 = n^2.

Reply:
No I think the total time complexity to do the flips would be $n^2$.

Reply:
Not the time complexity, the number of flips.

Reply:
Okay yes, the number of flips would be 2n.